src/main/java/pl/confitura/jelatyna/domain/Participant.java

```diff
@@ -22,4 +22,7 @@
     @Column(unique = true)
     private String email;
 
+    @OneToOne(mappedBy = "participant")
+    private User user;
```

Comment:
jak zabieraliśmy się za ten branch, to Participant już był.
Przyjęliśmy założenie, że Participant to będzie uczestnik - wypełnia formularz i już. Nie ma hasła i nie może się logować.

Wzięliśmy na warsztat security (i logowanie), bo wydawało mi się, że tu jest najwięcej decyzji które mają wpływ na architekturę rozwiązania. Liczyłem właśnie na dyskusję :)